changes.md

```diff
@@ -1,3 +1,52 @@
+## 📅 6 December 2024
+
+**Note:** Future updates to the changelog will be available at [github.blog/changelog](https://github.blog/changelog).
+
+### Image Preview Support
+
+You can now preview selected images directly from the file tree in Workspace.
```

Comment:
The wording here sounds like you can only preview certain images or maybe certain image types. Is there anything new you can do with the images? Can you create image assets in workspace, if so maybe the value is you can now review images in your workspace. Perhaps we link this to continued improvements from the previous changelog https://github.com/githubnext/copilot-workspace-user-manual/blob/28e07942a6392eb7164275cca0e20bb4c4afa6ac/changes.md#file-attachments

Reply:
I don't think we can link this to file attachement, this is really about seeing the asset in the editor

Reply:
How about: "You can now preview any image file you select from the file tree in Workspace."